src/lib/containers/evaluation_queues/EvaluationCard.tsx

```diff
@@ -1,21 +1,30 @@
 import React, { useEffect, useState } from 'react'
 import {
-  getEvaluation,
+  deleteEvaluation,
   getEvaluationPermissions,
   SynapseClientError,
 } from '../../utils/SynapseClient'
-import { capitalize } from 'lodash-es'
-import { Evaluation } from '../../utils/synapseTypes/Evaluation'
+import { Evaluation, EvaluationStatus } from '../../utils/synapseTypes'
 import { Button, Card, Col, Dropdown, Row } from 'react-bootstrap'
+import {
+  faClipboardCheck,
+  faEllipsisV,
+  faSyncAlt,
+} from '@fortawesome/free-solid-svg-icons'
 import { ErrorBanner } from '../ErrorBanner'
 import { CreatedOnByUserDiv } from './CreatedOnByUserDiv'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import { faEllipsisV } from '@fortawesome/free-solid-svg-icons'
 import { UserEvaluationPermissions } from '../../utils/synapseTypes/Evaluation/UserEvaluationPermissions'
+import { RequiredProperties } from '../../utils'
+
+export type ExistingEvaluation = RequiredProperties<
+  Evaluation,
+  'id' | 'etag' | 'createdOn' | 'contentSource' | 'ownerId' | 'status'
```

Comment:
This is a pretty slick type definition (marking the required field for an existing evaluation)